ngraph/test/files/paddlepaddle/gen_scripts/generate_squeeze.py

```diff
@@ -0,0 +1,37 @@
+#
+# squeeze paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle as pdpd
+
+data_type = 'float32'
+
+def squeeze(name : str, x, axes : list):
+    pdpd.enable_static()
+
+    with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
+        node_x = pdpd.static.data(name='x', shape=x.shape, dtype = data_type)
+        out = pdpd.fluid.layers.squeeze(node_x, axes=axes, name='squeeze')
+
+        cpu = pdpd.static.cpu_places(1)
+        exe = pdpd.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(pdpd.static.default_startup_program())
+
+        outs = exe.run(
+            feed={'x': x},
+            fetch_list=[out])             
+
+        saveModel(name, exe, feedkeys=['x'], fetchlist=[out], inputs=[x], outputs=[outs[0]])
+
+    return outs[0]
+
+def main():
+    data = np.random.rand(1, 3, 1, 4).astype(data_type)
+
+    squeeze("squeeze", data, [0, -2])
```

Comment:
this file is missing, only 'squeeze_null_axes' is present. Also 'squeeze' is not present in models.csv